real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -274,6 +274,56 @@ void steering_set(int angle) //TODO: should angle be an int or a long?
 }
 
 
+/** @brief Uses the steering center edge finder to detect the drive center.
+ *
+ *  This call may block for a while, but not more than the timeout based on
+ *  the search velocity and steering limits. We always search for the edge where
+ *  the sensor goes from no metal to metal detected so we aren't effected by
+ *  hysteresis of the system.
+ */
+int8_t steering_center() {
```

Comment:
We should standardize having brackets on their own line, but this can be changed in a new PR